util/plot_dram/dram_lat_mem_rd_plot.py

```diff
@@ -136,9 +136,9 @@ def is_pow2(num):
     xticks_labels = []
     for x in xticks_locations:
         if x < 1024:
-            xticks_labels.append("%d kB" % x)
+            xticks_labels.append("%d KiB" % x)
         else:
-            xticks_labels.append("%d MB" % (x / 1024))
+            xticks_labels.append("%d MiB" % (x / 1024))
```

Comment:
This should be further verified with the stats generation code in gem5. It might be right, but it could also be wrong.

Reply:
I looked further into this one and I think it should be okay to change. This script uses outputs from `configs/dram/lat_mem_rd.py`, which (among other things) generates base 2 memory range values. These values are printed into simout.txt, read in by `dram_lat_mem_rd_plot.py`, and ultimately used in xticks_labels.